src/utils/hooks/useDataViews.tsx

```diff
@@ -55,49 +56,54 @@ export const useDataViewsList = () => {
   return { error, loading, dataViewsList }
 }
 
-export const useDataViewsTable = ({ dataViewCode, apiQueries, filter }: DataViewTableProps) => {
+export const useDataViewFilterDefinitions = (dataViewCode: string) => {
   const { strings } = useLanguageProvider()
+  const [filterDefinitions, setFilterDefinitions] = useState<FilterDefinitions>()
   const [error, setError] = useState<ErrorResponse | null>(null)
-  const [loading, setLoading] = useState(false)
+  const [loading, setLoading] = useState(true)
+
+  useEffect(() => {
+    processRequest({
+      url: getServerUrl('dataViews', { dataViewCode, query: { first: '0' } }),
```

Comment:
`{ first: '0' } ` means we're not querying the database for data records at all, we'll just get the filter (and column) definitions in this first request.